### DISCUSSION TITLE
fluentui/react-northstar multi-select dropdown

### DISCUSSION THREAD

**Discussion by AMINSHAH** [microsoft/fluentui#16333] [Q&A] [answered]
I am working on Microsoft teams custom development. I need support regarding multi-select dropdown.
my question is 
how can I pass object array to dropdown items, if I am trying to do it will doesn't show me item list its just blank list
can anyone help please..

const **empList** = [

  { key: "1", text: "abc" },
  { key: "2", text: "xyz" },
];

 <Dropdown
            multiple
            search
            **items={empList}**
            placeholder="Start typing a name"
            getA11ySelectionMessage={getA11ySelectionMessage}
            noResultsMessage="We couldn't find any matches."
          />

**Comment by diabhoil** [comment, marked as answer]
Something like 

`const empList = [
{ key: "1", header: "abc" },
{ key: "2", header: "xyz" },
];`

should work, as the dropdown is waiting for [DropdownItemProps](https://fluentsite.z22.web.core.windows.net/0.52.0/components/dropdown/props#dropdown-item)
